src/gwt/src/org/rstudio/studio/client/workbench/views/files/FilesUpload.java

```diff
@@ -67,22 +67,27 @@ public void execute()
          new OperationWithInput<PendingFileUpload>() {
             public void execute(PendingFileUpload pendingUpload)
             {
-               // confirm overwrites if necessary
+               Boolean unzipFound = pendingUpload.getUnzipFound();
+               Boolean isZip = pendingUpload.getIsZip();
                FileUploadToken token = pendingUpload.getToken();
-               if (pendingUpload.getOverwrites().length() > 0)
+
+               // confirm unzip is installed
+               if (unzipFound == Boolean.FALSE && isZip == Boolean.TRUE)
                {
+                  // Warn user unzip is not installed
                   globalDisplay_.showYesNoMessage(
-                       MessageDialog.WARNING, 
-                       "Confirm Overwrite",
-                       confirmFileUploadOverwriteMessage(pendingUpload), 
-                       false, 
-                       completeFileUploadOperation(token, true), 
-                       completeFileUploadOperation(token, false), 
-                       false);
+                          MessageDialog.WARNING,
+                          "Unzip not found on PATH",
```

Comment:
Some users might not not be familiar with the concept of the PATH, so I would suggest slightly more plain language:

```suggestion
                          "unzip not found,
```

Reply:
I would recommend something really plain language like "Unable to open the archive because the archive opening tool (unzip) could not be found." Maybe something about checking with the sys admin if they want the archive to be extracted?

Reply:
Just saw Kevin's next message - really like that suggestion.

Reply:
<img width="370" alt="Screen Shot 2021-04-14 at 9 30 14 PM" src="https://user-images.githubusercontent.com/56326543/114815686-944bf700-9d6b-11eb-9dcc-de666df60022.png">